auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/model/AppUser.java

```diff
@@ -18,18 +18,18 @@ public class AppUser {
     private Long id;
 
 
-    @Column(nullable = false)
+    @Column(name="first_name",length = 50,nullable = false)
     private String firstName;
 
 
-    @Column(nullable = false)
+    @Column(name = "last_name",length = 50,nullable = false)
     private String lastName;
 
 
-    @Column(name = "email", nullable = false)
+    @Column(name = "email", length = 100 ,nullable = false, unique = true)
```

Comment:
If field name and property name in table are the same, then `name = "email"` should be redundant.